spec/support/currently_undocumented_cops.txt

```diff
@@ -1,3 +1,5 @@
+RuboCop::Cop::Bundler::DuplicatedGem
+RuboCop::Cop::Bundler::OrderedGems
```

Comment:
the introduction of a new namespace should be fine - [by default these will be categorized as "style".](https://github.com/codeclimate/codeclimate-rubocop/blob/master/lib/cc/engine/category_parser.rb) I think that probably sounds right... WDYT?